### PR TITLE
Add Czech Localization

### PR DESCRIPTION
**Pull request by isametry**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Do you know anyone that would be able to help review your localization?

**Comment by isametry** [comment]
Hmm, I don’t know… maybe @lemafor or @jezevec10 ?

File: Sources/KeyboardShortcuts/Localization/cs.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "Přidat zkratku";
+"press_shortcut" = "Stiskněte klávesy";
+"keyboard_shortcut_used_by_menu_item" = "Tuto zkratku nelze použít, protože ji už využívá položka „%@“ v nabídce.";
```
Comment (junkycoder) [inline Sources/KeyboardShortcuts/Localization/cs.lproj/Localizable.strings:3]: "Tuto zkratku nelze použít, protože je již využívána položkou „%@“." imho sound better in this context.

File: Sources/KeyboardShortcuts/Localization/cs.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+"keyboard_shortcut_used_by_system" = "Tuto zkratku nelze použít, protože už ji používá systém.";
```
Comment (junkycoder) [inline Sources/KeyboardShortcuts/Localization/cs.lproj/Localizable.strings:4]: 👌

File: Sources/KeyboardShortcuts/Localization/cs.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+"keyboard_shortcuts_can_be_changed" = "Většinu systémových zkratek můžete změnit v „Předvolby systému › Klávesnice › Zkratky“.";
```
Comment (junkycoder) [inline Sources/KeyboardShortcuts/Localization/cs.lproj/Localizable.strings:5]: 👌

**Review by junkycoder — COMMENTED** [review]
Already "looks good on dev", I just left few suggestions..

**Comment by YassinEldeeb** [comment]
I think @capaj can help.

**Review by jezevec10 — APPROVED** [review]
Both the original and the suggested translations can be used, no mistakes found.

**Review by capaj — APPROVED** [review]
LGTM, but suggestions by @junkycoder are even better in both places.

**Comment by isametry** [comment]
I also agree with the changes suggested by @junkycoder. In the first case, another possibility would be "Zadejte zkratku" or "Zadejte klávesy" – the latter is used by Apple in this context, and since this is a macOS package, staying consistent to the system would make the most sense I think.

press_shortcut

**Comment by sindresorhus** [comment]
> the latter is used by Apple in this context, and since this is a macOS package, staying consistent to the system would make the most sense I think.

👍

**Comment by sindresorhus** [comment]
@isametry You need to update your pull request with the changes ;)

**Comment by sindresorhus** [comment]
Thank you everyone :)

**Pull request merged**